libs/api/metadata-converter/src/lib/gn4/atomic-operations.ts

```diff
@@ -98,3 +100,19 @@ export const mapContact = (
     ...(phone && { phone }),
   }
 }
+
+export const mapKeywords = (thesauri: Thesaurus[], language: string) => {
+  const keywords = []
+  for (const thesaurusId in thesauri) {
+    const thesaurus = thesauri[thesaurusId]
+    for (const keyword of thesaurus.keywords) {
+      keywords.push({
+        label: selectTranslatedValue<string>(keyword, language),
+        type: getKeywordTypeFromKeywordTypeCode(thesaurus.theme),
+        ...(thesaurus.id && { thesaurus: { id: thesaurus.id } }),
```

Comment:
Just wondering if we shouldn't try to recover the `name` and `url` here as well? I see optionally `title` and `link` in the ES response.

Reply:
yes good point!

Reply:
done